types/nationalAvalancheCenter/schemas.ts

```diff
@@ -25,6 +25,13 @@ export function userFacingCenterId(input: AvalancheCenterID): string {
   return input === 'SNFAC' ? 'SAC' : input;
 }
 
+export const AvalancheCenterWebsiteSchema = {'https://nwac.us': 'NWAC'} as const;
```

Comment:
Couple small comments here:
 - this is not really a schema in that it doesn't describe the shape of any data
 - when you have to use `as` it usually means something went wrong - here I think you could more explicitly model this as a `Record<AvalancheCenterID, string>`
 - I don't think you ever want to look up the center based on its website, so the `reverseLookup` is likely not necessary

So this could become:

```ts
export const AvalancheCenterWebsites: Record<AvalancheCenterID, string> = {
  ['BAC']: 'https://bridgeportavalanchecenter.org/',
  ['BTAC']: 'https://bridgertetonavalanchecenter.org/',
  ['CBAC']: 'https://cbavalanchecenter.org/',
  ['CNFAIC']: 'https://www.cnfaic.org/',
  ['COAA']: 'https://www.coavalanche.org/',
  ['ESAC']: 'https://www.esavalanche.org/',
  ['FAC']: 'https://www.flatheadavalanche.org/',
  ['HPAC']: 'https://hpavalanche.org/',
  ['IPAC']: 'https://www.idahopanhandleavalanche.org/',
  ['KPAC']: 'https://kachinapeaks.org/',
  ['MSAC']: 'https://www.shastaavalanche.org/',
  ['MWAC']: 'https://www.mountwashingtonavalanchecenter.org/',
  ['NWAC']: 'https://nwac.us/',
  ['PAC']: 'https://payetteavalanche.org/',
  ['SAC']: 'https://www.sierraavalanchecenter.org/',
  ['SNFAC']: 'https://www.sawtoothavalanche.com/',
  ['TAC']: 'https://taosavalanchecenter.org/',
  ['WAC']: 'https://wallowaavalanchecenter.org/',
  ['WCMAC']: 'https://missoulaavalanche.org/',
};
```
Look-ups to get a website URL are just a map access.

Reply:
lol this is me being lazy and copying and pasting things without using my brain 😄